docs/classad-attributes/collector-classad-attributes.rst

```diff
@@ -1,74 +1,53 @@
 Collector ClassAd Attributes
 ============================
 
-
 :index:`Collector attributes<single: Collector attributes; ClassAd>`
-:index:`ActiveQueryWorkers<single: ActiveQueryWorkers; ClassAd Collector attribute>`
 
-``ActiveQueryWorkers``:
+:classad-attribute:`Collector attributes`
     Current number of forked child processes handling queries.
 
-:index:`ActiveQueryWorkersPeak<single: ActiveQueryWorkersPeak; ClassAd Collector attribute>`
-
-``ActiveQueryWorkersPeak``:
+:classad-attribute:`ActiveQueryWorkersPeak`
     Peak number of forked child processes handling queries since
     collector startup or statistics reset.
 
-:index:`PendingQueries<single: PendingQueries; ClassAd Collector attribute>`
-
-``PendingQueries``:
+:classad-attribute:`PendingQueries`
     Number of queries pending that are waiting to fork.
 
-:index:`PendingQueriesPeak<single: PendingQueriesPeak; ClassAd Collector attribute>`
-
-``PendingQueriesPeak``:
+:classad-attribute:`PendingQueriesPeak`
     Peak number of queries pending that are waiting to fork since
     collector startup or statistics reset.
 
-:index:`RecentDroppedQueries<single: RecentDroppedQueries; ClassAd Collector attribute>`
-:index:`DroppedQueries<single: DroppedQueries; ClassAd Collector attribute>`
+:index:`COLLECTOR_QUERY_WORKERS_PENDING`
 
-``DroppedQueries``:
+:classad-attribute:`RecentDroppedQueries`
     Total number of queries aborted since collector startup (or
     statistics reset) because ``COLLECTOR_QUERY_WORKERS_PENDING``
-
-:index:`COLLECTOR_QUERY_WORKERS_PENDING` exceeded, or
-    ``COLLECTOR_QUERY_MAX_WORKTIME``
+    exceeded, or ``COLLECTOR_QUERY_MAX_WORKTIME``
 
 :index:`COLLECTOR_QUERY_MAX_WORKTIME` exceeded, or client
```

Comment:
If we're moving index entries up to before the class-attribute tag, should this one move, too?